frontend/src/pages/Confirmation/reducer.js

```diff
@@ -72,13 +75,18 @@ const defaultState = fromJS({
   executingAdditionalActions: false,
   isListPermissionsRequiredFromApi: false,
   failedAction: {},
-  requestedPermissions: {}
+  requestedPermissions: {},
+  payloadValidationFailed : false
 });
 
 export default function confirmationReducer(state = defaultState, action) {
   switch (action.type) {
     case CONFIRMATION_REQUIRED:
       const { project, subproject, workflowitem } = action.payload;
+      const payloadValidationFailed = validate(action.intent, action.payload);
+      if(payloadValidationFailed) {
+        return state.merge({ payloadValidationFailed });
+      }
       return state.merge({
```

Comment:
rename `payloadValidationFailed` to `isPayloadValidationFailed`

Reply:
also change the return type of validate() to boolean. We do not need an object from validate